route/osrm/client.go

```diff
@@ -111,6 +111,13 @@ func (c *client) IgnoreEmpty(ignore bool) {
 	c.removeEmpty = ignore
 }
 
+// WithIgnoreEmpty configures if empty points should be ignored
```

Comment:
```suggestion
// WithIgnoreEmpty configures if empty points should be ignored.
```
So the linter passes.